google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -920,17 +920,7 @@ def default_acl
         #   policy = bucket.policy # API call
         #   policy_2 = bucket.policy # No API call
```

Comment:
This example raises the question of how to "force" a reload. Although, it does show the client does cache values to reduce the number of API calls.